src/main/java/com/gbtec/gameoflife/basic/GameOfLifeCommandProxy.java

```diff
@@ -0,0 +1,94 @@
+package com.gbtec.gameoflife.basic;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import org.springframework.messaging.simp.SimpMessagingTemplate;
+
+import lombok.Data;
+import lombok.RequiredArgsConstructor;
+import lombok.SneakyThrows;
+import lombok.extern.slf4j.Slf4j;
+
+/**
+ * Provides methods which allows to interact with the UI
+ */
+@RequiredArgsConstructor
+@Slf4j
+public abstract class GameOfLifeCommandProxy {
+
+    private static final String PRINTING_GOL_GENERATION_TOPIC_PATH = "/gol/generation";
```

Comment:
Do not abbreviate `gol`. To be most simple, we could just remove `/gol`, as the whole package and project is named `gameoflife`, anyway.